<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Connections</class>
 <widget class="QDialog" name="Connections">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Connections</string>
  </property>
  <widget class="QTabWidget" name="ConnectionTab">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>461</width>
     <height>351</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Search Connections</string>
    </attribute>
    <widget class="QPushButton" name="Remove">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>280</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string>Remove entry</string>
     </property>
     <property name="toolTipDuration">
      <number>999999999</number>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../icons.qrc">
       <normaloff>:/new/Icons/Icons/remove.svg</normaloff>:/new/Icons/Icons/remove.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
    <widget class="QLineEdit" name="quicksearch">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Quicksearch</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>20</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <item>
      <property name="text">
       <string>Scientists</string>
      </property>
      <property name="icon">
       <iconset resource="../icons.qrc">
        <normaloff>:/new/Icons/Icons/Scientists.svg</normaloff>:/new/Icons/Icons/Scientists.svg</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Computers</string>
      </property>
      <property name="icon">
       <iconset resource="../icons.qrc">
        <normaloff>:/new/Icons/Icons/Computers.svg</normaloff>:/new/Icons/Icons/Computers.svg</iconset>
      </property>
     </item>
    </widget>
    <widget class="QListWidget" name="Left">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>211</width>
       <height>241</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="Right">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>40</y>
       <width>211</width>
       <height>241</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>311</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Select an item in the left window to get all its connections</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Add Connections</string>
    </attribute>
    <widget class="QListWidget" name="left2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>211</width>
       <height>241</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="right2">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>40</y>
       <width>211</width>
       <height>241</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="leftsearch">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Scientist Search</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="rightsearch">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>20</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Computer Search</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Add">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>280</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string>Add new entry</string>
     </property>
     <property name="toolTipDuration">
      <number>999999999</number>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../icons.qrc">
       <normaloff>:/new/Icons/Icons/Add.svg</normaloff>:/new/Icons/Icons/Add.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>331</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Select an item from both windows to establish a connection</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections/>
</ui>
